.travis.yml

```diff
@@ -11,4 +11,5 @@ before_script:
   - ./.travis_do.sh download_sdk
 
 script:
+  - ./.travis_do.sh test_commits
```

Comment:
There is already one `set -e` in line 5 of `./.travis_di.sh`. Does it needs another `set -e`  to cover all failures?